### PR TITLE
Add validation of editor and secure previewToken, etc.

### PR DESCRIPTION
**Pull request by zimme**
Add validate-editor api and rename accessToken to previewToken.
This api endpoint will validate the params gotten from the storyblok editor mode and return the previewToken if it's valid

This update will validate the editor params using the new /validate-editor api, if you're a valid editor it will use storyblok-js-client to fetch the draft story based on the slug.

This commit also removes the storyblok accessToken from the frontend config and relies on the previewToken you get from the new /validate-editor api once validated.

This commit also contains some renames of variables, actions, mutations, getters, etc.

### CODE REVIEW

File: packages/vsf-storyblok-extension/index.js [hunk 1]
```diff
@@ -69,5 +70,25 @@ module.exports = ({ config, db }) => {
       error: false
     })
   })
+
+  api.get('/validate-editor', async (req, res) => {
+    const { spaceId, timestamp, token } = req.query
+
+    const validationString = spaceId + ':' + config.storyblok.previewToken + ':' + timestamp
```
Comment (benjick) [inline packages/vsf-storyblok-extension/index.js]: Do we wanna use `${spaceId}:${config.storyblok.previewToken}:${timestamp}` instead?

File: packages/vsf-storyblok-module/components/StoryblokMixin.ts [hunk 2]
```diff
@@ -1,32 +1,107 @@
...
+    script.onload = function onLoad () {
+      resolve()
+    }
+    document.getElementsByTagName('head')[0].appendChild(script)
+  })
+}
+
+function getStoryblokQuery (route) {
+  const queryString = route.fullPath.replace(route.path, '')
+  /* eslint-disable camelcase */
```
Comment (benjick) [inline packages/vsf-storyblok-module/components/StoryblokMixin.ts]:
```
const { _storyblok: id, _storyblok_c: c, _storyblok_tk: storyblok = {} } = qs.parse(queryString, { ignoreQueryPrefix: true })
const {space_id: spaceId, timestamp, token} = storyblok
```

File: packages/vsf-storyblok-module/components/StoryblokMixin.ts [hunk 3]
```diff
@@ -1,32 +1,107 @@
...
+
+    return { story }
+  },
+  async mounted () {
+    if (!this.story) {
+      this.fetchStory()
+    }
+
+    if (this.previewToken) {
+      // TODO: Make sure we don't load this multiple times
```
Comment (benjick) [inline packages/vsf-storyblok-module/components/StoryblokMixin.ts]: We actually do that in `loadScript`:

```
if (document.getElementById(id)) {
  resolve()
  return
}
```

Comment (zimme) [reply packages/vsf-storyblok-module/components/StoryblokMixin.ts]: Totally missed that 😄👍

**Pull request merged**